cmd/run/flags.go

```diff
@@ -182,4 +182,13 @@ func bindRunFlags(command *cobra.Command) {
 	flags.Uint32("listObjects-max-results", defaultConfig.ListObjectsMaxResults, "the maximum results to return in ListObjects responses")
 	util.MustBindPFlag("listObjectsMaxResults", flags.Lookup("listObjects-max-results"))
 	util.MustBindEnv("listObjectsMaxResults", "OPENFGA_LIST_OBJECTS_MAX_RESULTS", "OPENFGA_LISTOBJECTSMAXRESULTS")
+
+	flags.Bool("allow-writing-10-models", defaultConfig.AllowWriting10Models, "allow writing of schema 1.0 models")
+	util.MustBindPFlag("allowWriting10Models", flags.Lookup("allow-writing-10-models"))
+	util.MustBindEnv("allowWriting10Models", "OPENFGA_ALLOW_WRITING_1_0_MODELS", "OPENFGA_ALLOW_WRITING_10_MODELS", "OPENFGA_ALLOWWRITING10MODELS")
```

Comment:
Not sure the overrides are necessary here. The only reason we have some of them on others is because we introduced a breaking change at one point and wanted to maintain backward compatibility. Since this is a new flag and env variable, I say we just introduce `OPENFGA_ALLOW_WRITING_1_0_MODELS`.